scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java

```diff
@@ -6,21 +6,7 @@
 import java.util.List;
 import java.util.Map;
 
-import javax.persistence.CollectionTable;
-import javax.persistence.Column;
-import javax.persistence.ElementCollection;
-import javax.persistence.Embeddable;
-import javax.persistence.EmbeddedId;
-import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.Index;
-import javax.persistence.JoinColumn;
-import javax.persistence.Lob;
-import javax.persistence.ManyToOne;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import javax.persistence.Table;
-import javax.persistence.Transient;
+import javax.persistence.*;
```

Comment:
I use our [coding-rules configuration](https://github.com/ow2-proactive/coding-rules/blob/master/settings.jar) that uses the wildcard if there are more than 15 imports for the same package. I agree that wildcard may lead to confusion. We should keep that in mind for our new coding rules. I think Google profile don't use/allow wildcard by default.  
![capture d ecran de 2016-05-09 11-59-04](https://cloud.githubusercontent.com/assets/128898/15109839/bef8a4c8-15dd-11e6-8496-55c6ef48ae2c.png)